packages/backend/src/apps/paysg/common/schemas/payment.ts

```diff
@@ -0,0 +1,34 @@
+import z from 'zod'
+
+export const schema = z
+  .object({
+    id: z.string(),
+    payment_url: z.string(),
+    stripe_payment_intent_id: z.string(),
+    payment_qr_code_url: z.string(),
+    amount_in_cents: z.number(),
```

Comment:
These were `nullish()` before, but after thinking about it, those fields suddenly become unexpectedly null, our users' expectations will be subverted. Thus it should be an error and we should notify the user.

Have checked with the PaySG team that these fields are guaranteed to be nonnull.